src/api.js

```diff
@@ -49,20 +49,29 @@ let testPeople = [
   }
 ]
 
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
 export function getRandomPerson(filter = 'all') {
-  if ( process.env.REACT_APP_USE_TEST_DATA === "true") {
-    let data = testPeople.slice().sort(function(a,b) {return Math.random()-Math.random();}).slice(0,4);
-    return new Promise((res, rej) => res(data));
+  if (process.env.REACT_APP_USE_TEST_DATA === 'true') {
+    let data = testPeople.slice().sort(function(a, b) {
+      return Math.random() - Math.random();
+    }).slice(0, 4);
+    return new Promise(async function (res, rej) {
+      await sleep(1000);
```

Comment:
Why `sleep` when using test data? This doesn't seem to follow the `loading` flow, so I'm not clear on the benefit.